hal/ia32/exceptions.c

```diff
@@ -123,14 +125,15 @@ void hal_exceptionsDumpContext(char *buff, exc_context_t *ctx, int n)
 
 	n &= 0x1f;
 
-	__asm__ volatile
-	(" \
-		xorl %%eax, %%eax; \
-		movw %%ss, %%ax; \
-		movl %%eax, %0"
-	:"=r" (ss)
+	/* clang-format off */
+	__asm__ volatile(
+		"xorl %%eax, %%eax\n\t"
+		"movw %%ss, %%ax\n\t"
+		"movl %%eax, %0"
+	: "=r" (ss)
```

Comment:
simpler
```suggestion
	/* clang-format off */
	__asm__ volatile(
		"xorl %0, %0\n\t"
		"movw %%ss, %w0"
	: "=r" (ss)
```
and remove `eax` from the clobber list